### PR TITLE
Create a unique callback to handle error user info

### PR DESCRIPTION
**Pull request by BruunoMM**
Goal
---
Create a separate callback for the error event.

How to test
---
- Run test suite for both iOS and tvOS targets

### CODE REVIEW

File: Sources/Clappr_tvOS/Classes/Base/Player.swift [hunk 1]
```diff
@@ -222,21 +221,29 @@ open class Player: AVPlayerViewController {
 
     private func bindPlaybackEvents() {
         if let playback = core?.activePlayback {
+            var listenId = ""
```
Comment (paulogama14) [inline Sources/Clappr_tvOS/Classes/Base/Player.swift]: Same as above.

File: Sources/Clappr_tvOS/Classes/Base/Player.swift [hunk 2]
```diff
@@ -222,21 +221,29 @@ open class Player: AVPlayerViewController {
 
     private func bindPlaybackEvents() {
         if let playback = core?.activePlayback {
```
Comment (paulogama14) [inline Sources/Clappr_tvOS/Classes/Base/Player.swift]: Same as above.

File: Sources/Clappr_iOS/Classes/Base/Player.swift [hunk 3]
```diff
@@ -188,18 +188,26 @@ open class Player: BaseObject {
 
     private func bindPlaybackEvents() {
         if let playback = core?.activePlayback {
```
Comment (paulogama14) [inline Sources/Clappr_iOS/Classes/Base/Player.swift]: I know this is not from this PR, but we can change this if let to guard let, once we don't have code outside with a nullable core and we can remove one level of indentation.

File: Sources/Clappr_iOS/Classes/Base/Player.swift [hunk 4]
```diff
@@ -188,18 +188,26 @@ open class Player: BaseObject {
...
+            var listenId = ""
```
Comment (paulogama14) [inline Sources/Clappr_iOS/Classes/Base/Player.swift]: I think we don't need to share this variable. We can keep it inside for context. What do you think?

Comment (BruunoMM) [reply Sources/Clappr_iOS/Classes/Base/Player.swift]: We share so we don't have to define it every iteration and then again outside the for loop.

**Pull request merged**